### ISSUE TITLE
API doc problems of cross_entropy, softmax_with_cross_entropy

### ISSUE THREAD

**Issue by ranqiu92** [PaddlePaddle/Paddle#9070]
#### 单词拼写、遗漏和语法等问题未列出，修改时请注意
*请参考[API注释标准](https://github.com/PaddlePaddle/Paddle/blob/develop/doc/fluid/dev/api_doc_std_cn.md)！*
------
源文件：https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/layers/nn.py

- cross_entropy
http://www.paddlepaddle.org/docs/develop/api/en/fluid/layers.html#cross-entropy
  - 数据类型问题
  - 默认值问题（位置错误或缺失）
  -  格式问题
     1. \`ValueError\` -> ValueError
     2. Raise部分组织成一段话，或者分条列出时解决如下显示问题
![image](https://user-images.githubusercontent.com/31875089/37394358-862ce71e-27ae-11e8-9566-02728b6fa647.png)
  - 语言组织欠佳，请完善（如请将Returns部分组织成连贯的一句话）。

- softmax_with_cross_entropy
http://www.paddlepaddle.org/docs/develop/api/en/fluid/layers.html#sigmoid-cross-entropy-with-logits
  - 默认值问题（缺失或位置错误）
  - 格式问题 （如 Returns部分）

**Comment by shanyi15** [comment]
您好，此issue在近一个月内暂无更新，我们将于今天内关闭。若在关闭后您仍需跟进提问，可重新开启此问题，我们将在24小时内回复您。因关闭带来的不便我们深表歉意，请您谅解~感谢您对PaddlePaddle的支持! 
 Hello, this issue has not been updated in the past month. We will close it today for the sake of other user‘s experience. If you still need to follow up on this question after closing, please feel free to reopen it. In that case, we will get back to you within 24 hours. We apologize for the inconvenience caused by the closure and thank you so much for your support of PaddlePaddle Group!
